python/cudf/cudf/utils/ioutils.py

```diff
@@ -800,8 +800,13 @@
 comment : char, default None
     Character used as a comments indicator. If found at the beginning of a
     line, the line will be ignored altogether.
-na_values : list, default None
-    Values to consider as invalid
+na_values : scalar, str, or list-like, optional
+    Additional strings to recognize as nulls.
+    By default the following values are interpreted as
+    nulls: ‘’, ‘#N/A’, ‘#N/A N/A’, ‘#NA’, ‘-1.#IND’,
+    ‘-1.#QNAN’, ‘-NaN’, ‘-nan’, ‘1.#IND’, ‘1.#QNAN’,
+    ‘<NA>’, ‘N/A’, ‘NA’, ‘NULL’, ‘NaN’, ‘n/a’, ‘nan’,
+    ‘null’.
```

Comment:
These are non-default quote characters from copy pasting. Can you fix them?

Reply:
Fixed it